src/webargs/asyncparser.py

```diff
@@ -22,53 +21,12 @@ class AsyncParser(core.Parser):
     either coroutines or regular methods.
     """
 
-    async def _parse_request(
```

Comment:
This will be nice cleanup. 👍

Reply:
Aw, shucks. Thanks. 😊 

The refactor of the core parser is less obvious ("are we just shuffling around complexity?") but when you see how much this one changes and how much the diff is negative, it builds my confidence that this is really simplifying things.